app/src/test/java/io/apicurio/registry/noprofile/rest/v3/GroupsResourceTest.java

```diff
@@ -2808,7 +2913,7 @@ public void testCreateArtifactIntegrityRuleViolation() throws Exception {
 
         // Now try registering an artifact with both a valid and invalid ref
         data = new ByteArrayInputStream(artifactContent.getBytes(StandardCharsets.UTF_8));
-        var references = new ArrayList<io.apicurio.registry.rest.client.models.ArtifactReference>();
+//        var references = new ArrayList<io.apicurio.registry.rest.client.models.ArtifactReference>();
```

Comment:
Why is this commented out?

Reply:
It wasn't being used.  :)  I'll remove it.

Reply:
I didn't want to check the blame in case it was me who added it.  But it turns out it was Andrea so I'm off the hook.  rofl